package.json

```diff
@@ -155,7 +155,7 @@
 	"devDependencies": {
 		"@changesets/cli": "^2.26.1",
 		"@fluid-private/changelog-generator-wrapper": "file:tools/changelog-generator-wrapper",
-		"@fluid-tools/build-cli": "^0.29.0",
```

Comment:
Why any of these changes are here? These does not seem to be related to your change?

Reply:
Removed it!